pkg/scheduler/plugins/coscheduling/core/gang.go

```diff
@@ -124,19 +124,20 @@ func (gang *Gang) tryInitByPodConfig(pod *v1.Pod, args *config.CoschedulingArgs)
 
 	totalChildrenNum, err := strconv.ParseInt(pod.Annotations[extension.AnnotationGangTotalNum], 10, 32)
 	if err != nil {
-		klog.Errorf("pod's annotation totalNumber illegal, gangName: %v, value: %v",
+		klog.V(4).ErrorS(err, "pod's annotation totalNumber illegal, gangName: %v, value: %v",
 			gang.Name, pod.Annotations[extension.AnnotationGangTotalNum])
 		totalChildrenNum = int64(minRequiredNumber)
 	} else if totalChildrenNum != 0 && totalChildrenNum < int64(minRequiredNumber) {
-		klog.Errorf("pod's annotation totalNumber cannot less than minRequiredNumber, gangName: %v, totalNumber: %v,minRequiredNumber: %v",
+
+		klog.V(4).Infof("pod's annotation totalNumber cannot less than minRequiredNumber, gangName: %v, totalNumber: %v,minRequiredNumber: %v",
 			gang.Name, pod.Annotations[extension.AnnotationGangTotalNum], minRequiredNumber)
 		totalChildrenNum = int64(minRequiredNumber)
 	}
 	gang.TotalChildrenNum = int(totalChildrenNum)
```

Comment:
We encourage cleaner code, which of course includes improved logging.

```suggestion
var expectedTotalNum int64
s := pod.Annotations[extension.AnnotationGangTotalNum]
if s != "" {
   var err error
   expectedTotalNum, err = strconv.ParseInt(s)
   if err != nil {
      klog.V(4).ErrorS(err, "illegal annotation", "key", extension.AnnotationGangTotalNum, "value": s)
   }
}
if expectedTotalNum < int64(minRequiredNumber) {
   expectedTotalNum = int64(minRequiredNumber)
}
gang.TotalChildrenNum = int(expectedTotalNum)
```